pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
We are going to get a list of valid_courses (future PR), and check for the valid_course with the matching course_id.